lesson16/main.js

```diff
@@ -125,6 +127,28 @@ function createNewIconContainer(article) {
   return newIconContainer;
 }
 
+function createCommentIconContainer(article) {
+  const commentIconContainer = document.createElement("div");
+  commentIconContainer.classList.add("comment-container");
+  if (article.comment) {
+    const commentIcon = document.createElement("img");
+    commentIcon.classList.add("comment-icon");
+    commentIcon.src = "./img/comment.png";
+    commentIcon.width = "14";
+    commentIcon.height = "14";
+    commentIconContainer.appendChild(commentIcon);
+
+    const numOfComments = document.createElement("div");
+    numOfComments.classList.add("comment-num");
+    const numOfCommentProps = Object.keys(article.comment).length;

```

Comment:
Q. 
Is there a reason you used Object.keys?
I think it could be written this way
```suggestion
    const numOfCommentProps = article.comment.length;
```

Reply:
Thank you sae-san :)
It's simple !!
38b2da9614fe698a4d95d680141e9c5975a59473